get_github_coderabbit_review/coderabbit_fetcher/dependency_injection/decorators.py

```
@@ -0,0 +1,379 @@
+
+    This decorator adds metadata to help with dependency injection
+    and enables automatic constructor injection.
+
+    Args:
+        cls: Class to mark as injectable
+
+    Returns:
+        Decorated class
+    """
+    # Add injectable marker
+    setattr(cls, '_di_injectable', True)
```

Ruff (0.13.1)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)